source/manual/business-readiness-publish-changes.html.md

```diff
@@ -11,18 +11,13 @@ review_in: 3 months
 
 *If you are trying to update the content returned by the business readiness finder, see: [Update content for the business readiness finder](/manual/business-readiness-update-content.html). For more information about the finder itself, see: [EU exit business readiness finder](/manual/business-readiness-finder.html)*
 
-The full journey for the  [business readiness finder][business-readiness-finder] is made up of the following content items:
+The content for the start page ([`/business-uk-leaving-eu`][start-page]) can be updated in Mainstream Publisher as you would any other start page.
 
-1. The start page: [`/business-uk-leaving-eu`](https://www.gov.uk/business-uk-leaving-eu)
-2. The Q&A: [`/prepare-business-uk-leaving-eu`](https://www.gov.uk/prepare-business-uk-leaving-eu)
-3. The finder: [`/find-eu-exit-guidance-business`](https://www.gov.uk/find-eu-exit-guidance-business)
-4. The email signup page: [`/find-eu-exit-guidance-business/email-signup`](https://www.gov.uk/find-eu-exit-guidance-business/email-signup)
+The content item for the Q&A ([`/prepare-business-uk-leaving-eu`][q-and-a]) doesn't actually contain any details. The titles of the questions are defined in a YAML file in [finder frontend][finder-frontend]. However the body of the question, for example the options if the question is a checkbox, are read from the content item of the finder itself ([`/find-eu-exit-guidance-business`][business-readiness-finder]). Finder frontend searches the content item for a facet "key" that matches the question, e.g. `sector_business_area` to find the question content.
 
-The content for the start page can be updated in Mainstream Publisher as you would any other start page.
```

Comment:
As mentioned in the first commit, I would restore this section. This page is about publishing the finder, and the numbered list tells you what needs to be published.